### ISSUE TITLE
cp: -r wurde nicht angegeben, daher wird das Verzeichnis '/boot/' ausgelassen

### ISSUE THREAD

**Issue by ZarthKa** [Tomas-M/linux-live#144]
Hi Tomas,

I get an error-message I can't cope with.

`cp: -r wurde nicht angegeben, daher wird das Verzeichnis '/boot/' ausgelassen`

In English it would be like `cp: -r not provided, therefore directory '/boot/' left out` or similar.

My config looks like that:
̀̀
```
#!/bin/bash
# This is a config file for Linux Live Kit build script.
# You shouldn't need to change anything expect LIVEKITNAME

# Live Kit Name. Defaults to 'linux';
# This will be the name of the directory created on your CD/USB, which
# will contain everything including boot files and such.
# For example, Slax changes it to 'slax'
# Must not contain any spaces.
# If you change it, you must run ./tools/isolinux.bin.update script
# in order to update isolinux.bin for CD booting.
# If you do not need booting from CD (eg you're booting only from USB)
# then you can ignore recompiling isolinux.bin, just rename LIVEKITNAME
# and you're done.
LIVEKITNAME="Karstens_Manjaro_KDE_Live"

# Kernel file, will be copied to your Live Kit
# Your kernel must support aufs and squashfs. Debian Jessie's kernel is ready
# out of the box.
VMLINUZ=/boot/

# Kernel version. Change it to "3.2.28" for example, if you are building
# Live Kit with a different kernel than the one you are actually running
KERNEL=$(uname -r)

# List of directories for root filesystem
# No subdirectories are allowed, no slashes,
# so You can't use /var/tmp here for example
# Exclude directories like proc sys tmp
MKMOD="bin etc home lib lib64 opt root sbin srv usr var"

# If you require network support in initrd, for example to boot over
# PXE or to load data using 'from' boot parameter from a http server,
# you will need network modules included in your initrd.
# This is disabled by default since most people won't need it.
# To enable, set to true
NETWORK=false

# Temporary directory to store livekit filesystem
LIVEKITDATA=/tmp/$LIVEKITNAME-data-$$

# Bundle extension, for example 'sb' for .sb extension
BEXT=sb

# Directory with kernel .ko modules, can be different in some distros
LMK="lib/modules/$KERNEL"
̀`
```

I copied your scripts to /tmp/Tomas-M-linux-live-183e212/, the kernel is in /boot/.

When I start the build, like so:
```
[ktsn13 Tomas-M-linux-live-183e212]# ./build
Changing current directory to /tmp/Tomas-M-linux-live-183e212
Building intramfs image...
cp: -r wurde nicht angegeben, daher wird das Verzeichnis '/boot/' ausgelassen
```
the above is the result.

Can you give me a hint what the error message mentioned in the header means and how to solve it?

By the way, I'm using Manjaro Linux

```
karsten@ktsn13 Tomas-M-linux-live-183e212]$ uname -r
5.12.9-1-MANJARO
```
Thank you in advance for any given advice.

Regards

Karsten

**Comment by ZarthKa** [comment]
Same, same, but different:

From this [thread](https://superuser.com/questions/1602081/how-can-i-get-linux-live-kit-to-work), I think I found a solution. 

TLDR: They say, you have to 'see what is the correct one if you want to make the live with running kernel version run this command: `uname -r`' and change the vmlinuz on the config file to `vmlinuz-xx`

This did almost work for me except `uname -r`'s output was:
```
[karsten@ktsn13 ~]$ uname -r
5.12.9-1-MANJARO
```
and in /boot the vmlinuz files were:
```
[karsten@ktsn13 ~]$ ls /boot | grep vm
vmlinuz-5.10-x86_64
vmlinuz-5.12-x86_64
```

so I changed `VMLINUZ=/vmlinuz` to `VMLINUZ=/boot/vmlinuz-5.12-x86_64`

and it worked
